CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 ## Master
 
+##### Enhancements
+
+* Modified KeyboardToolbar to keep track of the next, back, and done buttons, so they can be enabled / disabled as needed. 
+* Added support to KeyboardScrollable to support an additional padding around a TextInputView when moving it into view
```

Comment:
Missing a period at the end of the sentence.

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/f0fb818e19adc3fe77eecb915f01f9ef479cc927